src/graphics/index-buffer.js

```diff
@@ -15,7 +15,7 @@ pc.extend(pc, function () {
      * @param {Number} numIndices The number of indices to be stored in the index buffer.
      * @param {Number} [usage] The usage type of the vertex buffer (see pc.BUFFER_*).
      */
-    var IndexBuffer = function (graphicsDevice, format, numIndices, usage) {
+    var IndexBuffer = function (graphicsDevice, format, numIndices, usage, initialData) {
         // Initialize optional parameters
         // By default, index buffers are static (better for performance since buffer data can be cached in VRAM)
         this.usage = usage || pc.BUFFER_STATIC;
```

Comment:
@guycalledfrank 
If usage parameter omitted, set initialData to this.usage.

Reply:
That might be a bit uselessly complicated, there are not too many arguments. And if we'd have more arguments, we could change them to be an object.

Reply:
I don't know what kind of use is supposed to be added in the future.
But many **usage** parameter of new IndexBuffer is omitted in [playcanvas-latest.js](https://code.playcanvas.com/playcanvas-latest.js).
So I thought that **usage** parameter is usually omitted.